src/routers/login/controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Request, Response } from 'express';
+import { Model } from 'mongoose';
+import { IUserModel, IUser } from "../../models/User";
+
+export const LoginController = (User: Model<IUserModel>) => { 
+    const handlerPost = (req: Request, res: Response) => (result: IUser) => {
+        const message: string = (result.password === req.body.password) ?
+            accessGrant(res) 
+            :
+            accessDenied(res);
+        res.send(message);
+    };
+    
+    const accessGrant = (res: Response) : string => {
+        res.status(201);
+        return 'granted access';
+    };
+
+    const accessDenied = (res: Response) : string => {
+        res.status(403);
+        return 'denied access';
+    }
+
+    const post = (req: Request, res: Response) => {
+        if (req.body.loginName && req.body.password) {
+            User.findOne({ 'loginName' : req.body.loginName })
+                .exec()
+                .then(handlerPost(req, res))
+                .catch(err => {
+                    console.log(err)
```

Comment:
Should we return here a response?

I remember that we have to return a 401 Unauthorized.

[403 forbidden vs 401 unauthorized](http://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses)